packages/client-utils/src/signing-smart-wallet-kit.ts

```diff
@@ -44,21 +62,25 @@ export const makeSigningSmartWalletKit = async (
   // Omit deprecated utilities
   const { storedWalletState: _, ...swk } = walletUtils;
 
+  const getLastUpdate = freeze(() => swk.getLastUpdate(address));
```

Comment:
this doesn't need to be bound separately from `query`. Please use `query.getLastUpdate` instead.